modules/nf-core/narfmap/align/tests/main.nf.test

```diff
@@ -213,8 +213,8 @@ nextflow_process {
                 input[0] = [
                     [ id:'test', single_end:false ], // meta map
                     [
-                        file(params.test_data['homo_sapiens']['illumina']['test_1_fastq_gz'], checkIfExists: true),
-                        file(params.test_data['homo_sapiens']['illumina']['test_2_fastq_gz'], checkIfExists: true)
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/fastq/test_1.fastq.gz', checkIfExists: true)
```

Comment:
Hmm, this one seems to have lost the comma at the end.

Reply:
Good catch! I'll try to figure out how it happened.